test/classes/phing/tasks/system/TouchTaskTest.php

```diff
@@ -106,4 +108,127 @@ public function testMappedFilelist()
         $tmpDir = $this->getProject()->getProperty('tmp.dir');
         $this->assertFileExists($tmpDir . '/touchtest');
     }
+
+    /**
+     * test millis attribute
+     */
+    public function testMillis()
+    {
+        // Don't run the test on 32-bit systems
+        if (PHP_INT_SIZE > 4) {
+            $this->executeTarget(__FUNCTION__);
+            $testFile = $this->getProject()->getProperty('tmp.dir') . '/millis-file';
+            $this->assertFileExists($testFile);
+
+            $this->assertEquals('December 31 1999 23:59:59', date("F d Y H:i:s", filemtime($testFile)));
```

Comment:
Do you mean something like this?
```php
$this->assertEquals('December 31 1999 23:59:59.000000', date("F d Y H:i:s.u", filemtime($testFile)));
```

Reply:
Seems unnecessary as filemtime() would always return the milliseconds as .000000.